docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -33,10 +33,21 @@ $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/stop
 $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/greeting
 
 <html>
-  <head><title>Error</title></head>
-  <body>503 - Service Unavailable</body>
+  <head>
+  ...
+  </head>
+  <body>
+    <div>
+      <h1>Application is not available</h1>
+      ...
+    </div>
+  </body>
 </html>
 ----
++
+NOTE: Depending on when OpenShift removes the pod after you invoke the /api/stop endpoint, you may initially see a 404 error code. If continue to invoke the /api/greeting endpoint, you will see the `Application is not available` page after OpenShift removes the pod.
+
+It is not accurate as following stop request by greeting immediately will most likely result in 404 error as OpenShift might not have noticed the issue yet. Once the OpenShift takes the pod out, the expected outcome is also different from the one specified.
```

Comment:
I don't think that this was meant to be here :)

Reply:
Nice, catch. sorry, copy/paste error.